workbench-core/infrastructure/src/workbenchAppRegistry.test.ts

```diff
@@ -59,17 +59,17 @@ describe('WorkbenchAppRegistry Test', () => {
 
     template.hasResourceProperties('AWS::ServiceCatalogAppRegistry::AttributeGroupAssociation', {
       Application: {
-        'Fn::GetAtt': ['AppRegistry968496A3', 'Id']
+        'Fn::GetAtt': ['TestStackApplicationF250E570', 'Id']
```

Comment:
`F250E570` is really a constant which can be used reliably as a literal here? Same applies to similar on line 65.

Reply:
Yes @DaveMadsen we can use these literals here, these don't change unless the code is updated